src/NServiceBus.RabbitMQ/Configuration/DelayedDeliverySettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Transport.RabbitMQ
+{
+    using Configuration.AdvanceExtensibility;
+    using Settings;
+
+    /// <summary>
+    /// TODO: Fill this out
+    /// </summary>
+    public class DelayedDeliverySettings : ExposeSettings
+    {
+        internal DelayedDeliverySettings(SettingsHolder settings) : base(settings) { }
+
+        /// <summary>
+        /// TODO: Fill this out
+        /// </summary>
+        public void DisableTimeoutManager()
```

Comment:
@WojcikMike fluent apis are discouraged afaik. NSB v4 had a fluent api that was dropped. @bording do you remember what was the thought on fluent apis that steered away from using it?

I suggested another option (below), but we chose not to go that path. We could review configuration approach again if needed.

```c#
var transport = endpointConfig.UseTransport<RabbitMQTransport>().ConnectionString("...");
transport.DelayedDelivery(config => 
{
  config.DisableTimeoutManager();
  config.AllEndpointsSupportNativeDelays();
});

The approach above could be somewhat more aligned with other configurations we see in Core nowadays. Thoughts?